publish.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/sh
```

Comment:
Rename file to something better. Don't forget to refactor where this file is suppose to be called from. chmod +x????